2025/social-wg.html

```diff
@@ -160,7 +160,7 @@ <h2>Motivation and Background</h2>
         <p>
           In the past few years, the Social Web Incubator Community Group acted as a
           maintainer for Social Web related specifications published by W3C.
-          This Working Group would maintain the Recommendations and Notes in the
+          This Working Group will take over the maintenance of the Recommendations and Notes in the
```

Comment:
```suggestion
          This Working Group will maintain Recommendations and Notes in the
```
minor edit, pick a more "current" phrasing